### PR TITLE
Translate eloquent.md

### PR DESCRIPTION
**Pull request by neighborhood999**
#287
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48395186%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48395198%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48395240%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48396438%22%2C%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48399726%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%204e86d8b4d56bc8247bc4240dedd31f722ac66ded%20eloquent.md%20644%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48395240%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22local%20%5Cu7ffb%5Cu6210%2A%2A%5Cu5c40%5Cu90e8%2A%2A%5Cu6216%2A%2A%5Cu5340%5Cu57df%2A%2A%5Cu5427%5Cuff1f%22%2C%20%22created_at%22%3A%20%222015-12-24T04%3A20%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cu6211%5Cu89ba%5Cu5f97%5Cu7ffb%5Cu6210%2A%2A%5Cu5c40%5Cu90e8%2A%2A%5Cu597d%5Cuff0c%20%40jigsawye%20%5Cu4f60%5Cu89ba%5Cu5f97%5Cu5462%5Cuff1f%22%2C%20%22created_at%22%3A%20%222015-12-24T05%3A18%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10325111%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/neighborhood999%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cu90a3%5Cu5c31%5Cu5c40%5Cu90e8%5Cu5427%5Cuff01%22%2C%20%22created_at%22%3A%20%222015-12-24T07%3A18%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20eloquent.md%3AL564-592%22%7D%2C%20%22Pull%204e86d8b4d56bc8247bc4240dedd31f722ac66ded%20eloquent.md%20573%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48395198%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu6240%5Cu4ee5%5Cu4f60%5Cu53ef%5Cu4ee5%5Cu81ea%5Cu7531%5Cu5730%5Cu5728%20Laravel%20%5Cu61c9%5Cu7528%5Cu7a0b%5Cu5f0f%5Cu7684%20%60app%60%20%5Cu76ee%5Cu9304%5Cu4e2d%5Cu5efa%5Cu7acb%5Cu81ea%5Cu5df1%5Cu7684%20%60Scopes%60%20%5Cu8cc7%5Cu6599%5Cu593e%5Cu3002%22%2C%20%22created_at%22%3A%20%222015-12-24T04%3A18%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20eloquent.md%3AL504-515%22%7D%2C%20%22Pull%204e86d8b4d56bc8247bc4240dedd31f722ac66ded%20eloquent.md%20564%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/laravel-taiwan/docs/pull/306%23discussion_r48395186%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu5957%5Cu7528%5Cu7bc4%5Cu570d%5Cu81f3%5Cu7d66%5Cu5b9a%5Cu7684%20Eloquent%20%5Cu67e5%5Cu8a62%5Cu69cb%5Cu9020%5Cu5668%5Cu3002%22%2C%20%22created_at%22%3A%20%222015-12-24T04%3A17%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8567270%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jigsawye%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20eloquent.md%3AL492-499%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 4e86d8b4d56bc8247bc4240dedd31f722ac66ded eloquent.md 564'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/306#discussion_r48395186'>File: eloquent.md:L492-499</a></b>
- <a href='https://github.com/jigsawye'></a> 套用範圍至給定的 Eloquent 查詢構造器。
- [ ] <a href='#crh-comment-Pull 4e86d8b4d56bc8247bc4240dedd31f722ac66ded eloquent.md 573'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/306#discussion_r48395198'>File: eloquent.md:L504-515</a></b>
- <a href='https://github.com/jigsawye'></a> 所以你可以自由地在 Laravel 應用程式的 `app` 目錄中建立自己的 `Scopes` 資料夾。
- [ ] <a href='#crh-comment-Pull 4e86d8b4d56bc8247bc4240dedd31f722ac66ded eloquent.md 644'></a> &nbsp;<b><a href='https://github.com/laravel-taiwan/docs/pull/306#discussion_r48395240'>File: eloquent.md:L564-592</a></b>
- <a href='https://github.com/jigsawye'></a> local 翻成**局部**或**區域**吧？
- <a href='https://github.com/neighborhood999'></a> 我覺得翻成**局部**好， @jigsawye 你覺得呢？
- <a href='https://github.com/jigsawye'></a> 那就局部吧！

<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/306?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/laravel-taiwan/docs/pull/306?approve=1'></a>&nbsp;<a href='https://github.com/laravel-taiwan/docs/pull/306'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by neighborhood999** [comment]
#306

File: eloquent.md [hunk 1]
```diff
@@ -492,7 +492,7 @@ Writing a global scope is simple. Define a class that implements the `Illuminate
     class OldScope implements Scope
     {
         /**
-         * Apply the scope to a given Eloquent query builder.
+         * 分配給定的範圍到給定的 Eloquent 搜尋構造器。
```
Comment (jigsawye) [inline eloquent.md]: 套用範圍至給定的 Eloquent 查詢構造器。

File: eloquent.md [hunk 2]
```diff
@@ -504,11 +504,11 @@ Writing a global scope is simple. Define a class that implements the `Illuminate
         }
     }
 
-There is not a predefined folder for scopes in a default Laravel application, so feel free to make your own `Scopes` folder within your Laravel application's `app` directory.
+在預設的 Laravel 應用程式裡沒有預先定義範圍（scpoes）的資料夾，所以你可以自由地建立自己的 `Scopes` 資料夾在你的 Laravel 應用程式的 `app` 目錄下。
```
Comment (jigsawye) [inline eloquent.md]: 所以你可以自由地在 Laravel 應用程式的 `app` 目錄中建立自己的 `Scopes` 資料夾。

File: eloquent.md [hunk 3]
```diff
@@ -564,28 +564,28 @@ Eloquent also allows you to define global scopes using Closures, which is partic
...
-If you would like to remove several or even all of the global scopes, you may use the `withoutGlobalScopes` method:
+如果你想要移除幾個甚至所有的全域範圍，你可以使用 `withoutGlobalScopes` 方法：
 
     User::withoutGlobalScopes()->get();
 
     User::withoutGlobalScopes([FirstScope::class, SecondScope::class])->get();
 
 <a name="local-scopes"></a>
-### Local Scopes
+### 查詢範圍
```
Comment (jigsawye) [inline eloquent.md]: local 翻成**局部**或**區域**吧？

Comment (neighborhood999) [inline eloquent.md]: 我覺得翻成**局部**好， @jigsawye 你覺得呢？

Comment (jigsawye) [inline eloquent.md]: 那就局部吧！

**Pull request merged**